lumigator/python/mzai/sdk/tests/conftest.py

```diff
@@ -1,9 +1,12 @@
+import sys
+sys.path.append("../schemas")
```

Comment:
do we still need this sys path append? this should be covered by the editable install

Reply:
The editable install has the issue that creates a package dependency. Even though this dependency is resolved in the dev environment, the build environment won't include it (or at least I didn't manage to do it). What I can do is include the schemas module as a source folder, but this means that is part of the lumigator-sdk _package_, so packaging-wise there's no separate schemas package.

Tests then have the issue of where they should get the schemas, since they're not a package on their own. This is why the sys path needs to be tweaked in pytest. We could solve this by packaging the schemas independently and setting/checking that the sdk and schemas package versions are exactly the same. Otherwise we need to resort to this, afaict.

Reply:
We will handle schemas as a separate package, and simplify the dependencies.